railties/lib/rails/generators/rails/app/templates/rubocop.yml.tt

```diff
@@ -1,6 +1,9 @@
 # Omakase Ruby styling for Rails
 inherit_gem: { rubocop-rails-omakase: rubocop.yml }
 
+AllCops:
+  TargetRubyVersion: <%= ENV["RBENV_VERSION"] || ENV["rvm_ruby_string"] || RUBY_ENGINE_VERSION %>
```

Comment:
Rubocop should be using the value in `.ruby-version` by default, which we already generate for apps: https://github.com/rails/rails/blob/d8f68f9d558be9420d9432dcfcb5b80a2b3cb161/railties/lib/rails/generators/rails/app/templates/ruby-version.tt

Reply:
Indeed it should! I'll go back and double-check why I was seeing 2.7 syntax enforcement.

Reply:
Yep, reverting. It was due to not having a `.ruby-version` in generated plugins' `test/dummy`. See `Rails::PluginBuilder#test_dummy_clean`.